gamemodes/terrortown/entities/weapons/weapon_tttbasegrenade.lua

```diff
@@ -305,3 +312,46 @@ function SWEP:OnRemove()
 		RunConsoleCommand("use", "weapon_ttt_unarmed")
 	end
 end
+
+if CLIENT then
+	local draw = draw
+	local TryT = LANG.TryTranslation
+	local hudTextColor = Color(255, 255, 255, 180)
+
+	---
+	-- @ignore
+	function SWEP:DrawHUD()
+		if self.HUDHelp then
+			self:DrawHelp()
+		end
+
+		if self:GetPin() and self:GetPullTime() > 0 then
+			local client = LocalPlayer()
+
+			local x = ScrW() / 2.0
```

Comment:
For some reason it is fast to multiply with 0.5 instead of deciding by 2. Don't ask me why